src/ms.tex

```diff
@@ -284,7 +284,49 @@ \subsection{Connections with Observatories and Missions}
 \secunfilled
 
 Cherenkov Telescope Array
-\secauthor{Axel Donath, Max Noethe}
+\secauthor{Axel Donath, Maximilian Nöthe}
+
+The Cherenkov Telescope (CTA) will be the next generation very-high-energy
+gamma-ray observatory.
+It will improve over the current generation of imaging atmospheric Cherenkov telescopes (IACTs)
+by a factor of five to ten in sensitivity and it will be able to observe the whole sky from a combination of two sites:
+a northern site in La Palma, Spain, and a southern one in Paranal, Chile.
+CTA will be able to observe gamma rays in the energy range between $20\,\mathrm{GeV}$ and $300\,\mathrm{GeV}$
```

Comment:
Should be 300 **TeV** here...

Reply:
Thanks. Tried to use siunitx but that seems to be incompatible with the template. Then copy/paste...